spec/Elf.3d

```diff
@@ -0,0 +1,461 @@
+/*
+  Copyright (c) Microsoft Research
+  SPDX-License-Identifier: Apache-2.0
+  From: https://github.com/project-everest/everparse/blob/master/src/3d/tests/ELF.3d
```

Comment:
Would it be possible to pull in this file via a submodule instead of copying it?   (Submodules make pulling in updates much easier than copying)